packages/special-pages/pages/errorpage/src/index.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>$TITLE$</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="content-container">
+        <div class="error-container">
+            <div class="header-container">
+                <h1 class="error-header">$HEADER$</h1>
+            </div>
+            <div class="description-container">
+                <p class="error-description">$ERROR_DESCRIPTION$</p>
+                <!-- <p class="error-code">Error code: $ERROR_CODE$</p> -->
+            </div>
```

Comment:
@mallexxx can you give some examples of the kinds of strings that get placed here? Are they all well-known to us, and do not contain dynamic/user data?

I'm trying to figure out what risk (if any) there is here - for example, if a user was able to create a situation where a malious payload made it into the HTML via these replacements?

Reply:
$TITLE$ - title of the failing page - currently it's "Oops" - provided from our localization values
$HEADER$ - "Page could not be loaded" - provided from our localization values
$ERROR_DESCRIPTION$ - error localized description provided by WebKit
$ERROR_CODE$ - error code (int) provided by WebKit

All the values are processed by Apple's NSAttributedString text-to-html converter before insertion which replaces html-invalid characters with escape codes (like &lt; &gt; &amp;)

Reply:
Currently there's no arbitrary user values inserted @shakyShane